template/src/components/virtual-background/useVB.tsx

```diff
@@ -189,6 +178,33 @@ const VBProvider: React.FC = ({children}) => {
     fetchData();
   }, []);
 
+  const applyVirtualBackgroundToMainView = async (
+    config: VirtualBackgroundConfig,
+  ) => {
+    //@ts-ignore
+    const localVideoTrack = RtcEngineUnsafe?.localStream?.video;
+    //  mainViewProcessor && (await mainViewProcessor.disable()); // Disable the old processor
+    localVideoTrack
+      ?.pipe(mainViewProcessor)
+      .pipe(localVideoTrack?.processorDestination);
+    mainViewProcessor.setOptions(config);
+    await mainViewProcessor.enable();
+  };
+
+  // Function to apply virtual background to the preview view
+  const applyVirtualBackgroundToPreviewView = async (
+    config: VirtualBackgroundConfig,
+  ) => {
```

Comment:
is there two separate tracks for previewView and main view ? then which 0f it is being used for the subcomponent `VideoPreview`